doc/Known_issues/CredSSP_error_bestiary.md

```diff
@@ -0,0 +1,28 @@
+# I get authentication error "CredSSP encryption Oracle remediation" when trying to RDP into a Windows machine from another Windows machine
```

Comment:
MKDocs (documentation generator) uses the first line of a document as page title, leading to this (see the long page name in the panel on the left):

![image](https://user-images.githubusercontent.com/9220158/48894984-4eadcd00-ee44-11e8-9b11-c83fa1f4a2c5.png)

Could you please transform this page to:

```
# Title
## I get authentication error ...
### Changing the options in the server itself
### Changing the options on the client
```

Reply:
ok